site/content/en/references/kustomize/kustomization/helmcharts/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Helm Charts"
+linkTitle: "Helm Charts"
+type: docs
+weight: 6
+description: >
+Helm chart inflation generator
+---
+
+[kustomize builtins]: https://kubectl.docs.kubernetes.io/references/kustomize/builtins/#_helmchartinflationgenerator_
+[Helm support long term plan]: https://github.com/kubernetes-sigs/kustomize/issues/4401
+
+## Helm Chart Inflation Generator
+
+Kustomize has limited support for helm chart inflation through the `helmCharts` field.
+You can read a detailed description of this field in the docs about [kustomize builtins].
+
+## Long term support
+
+The helm chart inflation generator in kustomize is intended to be a limited subset of helm features to help with
+getting started with kustomize, and we cannot support the entire helm feature set.
+
+### The current builtin
+For enhancements to the helm chart inflation generator feature, we will only support the following changes:
+
+- bug fixes
+- critical security issues
+- additional fields that are analogous to flags passed to `helm template`
+
+We will not add support for:
+- private repository or registry authentication
+- OCI registries
+- other large features that increase the complexity and security of the feature
```

Comment:
```suggestion
- other large features that increase the complexity of the feature and/or have significant security implications
```

(as you said above, we will fix security issues)

Reply:
done